### ISSUE TITLE
[PLUGIN REVIEW] pysap-etomo

### ISSUE THREAD

**Issue by sfarrens** [CEA-COSMIC/pysap#168]
## Plug-In Description

**Which type of plug-in do you want reviewed?**
- [ ] A new plug-in that is not hosted on CEA-COSMIC
- [X] An existing plug-in that is hosted on CEA-COSMIC

**New plug-in details**
- Repository URL: https://github.com/CEA-COSMIC/pysap-etomo
- Where is example data hosted:  *e.g.* ftp://MYFTP.com
- What is the purpose of this plugin? What is the application?

**Existing plugin details**
- Plugin name: `pysap-etomo`

## PySAP Plug-In Integration Checklist

> In order for a new plug-in to be integrated into the PySAP suite the following points should be checked. Once all points are chedked, the plug-in should be tagged and released on PyPi in order to be included in the next PySAP release.

- [x] Provides tools for applications not already covered by an existing PySAP plug-in (_e.g._ a new imaging domain)
- [x] Makes appropriate use of PySAP and ModOpt tools (_i.e._ fits into PySAP framework and does not reinvent the wheel) 
- [x] API consistent with existing PySAP plugins (_i.e._ naming conventions and architecture are consistent with other plug-ins)
- [x] Any new dependencies justified
- [x] Unit tests implemented in a consistent way
- [x] Sufficient coverage of unit tests
- [x] Docstrings implemented in a consistent way
- [x] Docstrings clearly describe new functions/classes (_i.e._ is it clear for the user what all the functions and classes do)
- [x] Example scripts and data available for testing

**Comment by sfarrens** [comment]
Hi @philouc and @chaithyagr, as you probably have seen @Biagig has already [tagged a release of pysap-etomo](https://github.com/CEA-COSMIC/pysap-etomo/releases/tag/v0.0.1). We should be all set to include this into the next release of PySAP, but since we still have some time to be thorough, I would kindly request (if you can find some time) that you have a quick look at the [pysap-etomo repo](https://github.com/CEA-COSMIC/pysap-etomo) and check any of the points listed in the description above. If you are happy that that said point is satisfied, please tick the box in the description. I will check everything unticked next week.

**Comment by sfarrens** [comment]
@Biagig @philouc I have opened a [PR on pysap-etomo](https://github.com/CEA-COSMIC/pysap-etomo/pull/5) to resolve some issues I encountered when attempting to integrate this plug-in into PySAP.

I have also opened a [minor issue regarding one of the example scripts](https://github.com/CEA-COSMIC/pysap-etomo/issues/6).

I believe we will need to do some more work to improve this plug-in going forward, but I believe if we can resolve this PR and issue I can finalise the integration and make the next PySAP release.
